package.json

```diff
@@ -49,17 +49,19 @@
     "coveralls": "^2.11.12",
     "eslint": "^3.3.1",
     "eslint-config-airbnb": "^10.0.1",
-    "eslint-config-airbnb-base" : "^5.0.2",
+    "eslint-config-airbnb-base": "^5.0.2",
     "eslint-plugin-import": "^1.13.0",
     "eslint-plugin-jsx-a11y": "^2.1.0",
     "eslint-plugin-react": "^6.1.1",
     "istanbul": "^0.4.4",
+    "jszip": "^3.1.2",
```

Comment:
Yea because archives doesn't unarchive :D so it's still needed for the tests

Sent from my iPhone

> On 26 Aug 2016, at 10:48, Florian Motlik <notifications@github.com> wrote:
> 
> In package.json:
> 
> >      "eslint-plugin-import": "^1.13.0",
> >      "eslint-plugin-jsx-a11y": "^2.1.0",
> >      "eslint-plugin-react": "^6.1.1",
> >      "istanbul": "^0.4.4",
> > +    "jszip": "^3.1.2",
> I jszip still necessary here?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>